src/main/java/uk/gov/hmcts/reform/sscs/TribunalsCaseApiApplication.java

```diff
@@ -189,20 +187,20 @@ public DocmosisPdfGenerationService docmosisPdfGenerationService(
     @Value("${gov.uk.notification.api.testKey}")
     private String testApiKey;
 
-    @Bean
-    public ServletListenerRegistrationBean<ServletContextListener> appInsightsServletContextListenerRegistrationBean(
-        ApplicationInsightsServletContextListener applicationInsightsServletContextListener) {
-        ServletListenerRegistrationBean<ServletContextListener> srb =
-            new ServletListenerRegistrationBean<>();
-        srb.setListener(applicationInsightsServletContextListener);
-        return srb;
-    }
-
-    @Bean
-    @ConditionalOnMissingBean
-    public ApplicationInsightsServletContextListener applicationInsightsServletContextListener() {
-        return new ApplicationInsightsServletContextListener();
-    }
+    //    @Bean
```

Comment:
is this config coming back or can it be deleted?

Reply:
good question, I forgot about this I was marking other things with TODO: SB3 but this slipped through the cracks. I'll take a look